crates/config/src/lib.rs

```diff
@@ -1092,6 +1099,10 @@ impl Config {
         &self,
         maybe_alias: &str,
     ) -> Option<Result<Cow<'_, str>, UnresolvedEnvVarError>> {
+        if let Ok(Some(endpoint)) = mesc::get_endpoint_by_query(maybe_alias, Some("foundry")) {
```

Comment:
this will prefer mesc aliases over ones in foundry.toml, should we flip it to make it easier to override?

Reply:
changed the order; foundry alias first, mesc second